Added Homework1,2,3, Homеwork 6

NatalliaYe opened the pull request:
Homework 2.
1 -Name, lastName, profession.
2 - Declared and initialized variables. 
3 - imported scanner, printed out result of all studied string methods
(cherry picked from commit 92a894374d2d0a3cbf18c64322ead760912b3d76)

Homework 3. Added 4 tasks(remote control and others)
Homework 4 Added fixed error.
Homework 6. Creating file .txt; reading from file]

NatalliaYe committed:
Added Homework part1 -Name, lastName, profession.
Homework part2 - Declared and initialized variables.
Homework part3 - imported scanner, printed out result of all studied string methods

(cherry picked from commit 92a894374d2d0a3cbf18c64322ead760912b3d76)

NatalliaYe committed:
Homework 3. Added 4 tasks

Fulcrum81 commented on src/anotherHomework3.java:
```diff
@@ -0,0 +1,20 @@
+        int sum = 0;
+        if (x >= 0) {
+            for (int i = 0; i <= x; i++) {
+                sum += 1;
```
эквивалентно sum++

Fulcrum81 replied:
К слову это не соответствует ТЗ. В твоем случае он будет прибавлять единицу, т.е. по итогу посчитает просто до самого числа, а надо чтобы он сумму всех чисел посчитал

Fulcrum81 commented on src/anotherHomework3.java:
```diff
@@ -0,0 +1,20 @@
+            }
+        }else {
+            for (int i = 0; i >= x; i--) {
+                sum += 1;
```
число будет положительным, это некорректно

Fulcrum81 commented on src/anotherHomework4.java:
```diff
@@ -0,0 +1,40 @@
+            if (scanner.hasNextInt()) {
+                int channelNumber = scanner.nextInt();
+
+                switch (channelNumber) {
```
Здесь switch избыточен, обрати внимание, что для всех кейсов, кроме 0 поведение одинаковое.
Подумай, как упростить

NatalliaYe committed:
Homework 4.
-Made changes on Homework3 - in anotherHomework3 and anotherHomework4.
-Committed New Homework4 (Commode,Clothes,Main,MyInfo)

NatalliaYe committed:
Homework 4.Added  fixed error
-Made changes to Commode,Clothes,Main

NatalliaYe committed:
Homework 6.  Creating file .txt; reading from file
